.github/workflows/ci.yml

```diff
@@ -46,9 +79,9 @@ jobs:
             ${{ env.XDG_CACHE_HOME }}/vcpkg/archives
             ${{ env.LOCALAPPDATA }}\vcpkg\archives
             ${{ env.APPDATA }}\vcpkg\archives
```

Comment:
To cache the tools that are downloaded directly, you can add this to the paths.  As I said, the time saved with this method might not be much as we are just downloading from a different GitHub server. Windows cache decompression is also much slower than on Ubuntu.
https://github.com/actions/cache/issues/442

```
            ${{ env.APPDATA }}\vcpkg\archives
            ~/llvm
            ~/cmake
            ~/ninja
```

Reply:
Ok, you're right; I missed that part.